docs/configuration.md

```diff
@@ -553,6 +559,29 @@ A fixed slot (mirroring `bid_strategy.order_book_top`) can be defined by setting
 
 When not using orderbook (`ask_strategy.use_order_book=False`), the price at the `ask_strategy.price_side` side (defaults to `"ask"`) from the ticker will be used as the sell price.
 
+### Market order pricing
+
+When using market orders, prices should be configured to use the "correct" side of the orderbook to allow realistic pricing detection.
```

Comment:
"When using market orders with orderbook enabled, prices should be configured ..."?

Reply:
no? unless i misunderstand your intend.

it does not matter if you use orderbook or not, it's the side that matters.


ticker-prices (ask / bid values from the `fetch_ticker()` result do correspond (should correspond) to the best orderbook bid / ask prices.
The real difference between using ticker and orderbook is speed (at least i feel that way).

orderbook is usually a "cheaper" call with less impact on ratelimits (at least according to many exchange documentations) - but no matter if using ticker or orderbook - the price-side is applied and should be reversed for market orders.

Now for market orders, you could drive this configuration further and use the 2nd or 3rd slot in the oderbook assuming you're pushing past the first slot - but that'll too strongly depend on markets (and your stake-amount) - so i would rather not complicate this section too much.

Reply:
>it does not matter if you use orderbook or not, it's the side that matters.

I just do not remember does it have effect when orderbook is not enabled (and ticker data is therefore used)

okay, not "with orderbook enabled", forget it...